guides/managing-current-user.md

```diff
@@ -63,7 +63,7 @@ invoked with a query param where the `me` param is present.
 // app/services/current-user.js
 import Ember from 'ember';
 
-const { inject: { service }, isEmpty, RSVP } = Ember;
```

Comment:
`RSVP` is used below as `Ember.RSVP` actually - I think it would be better to keep it here and change the usage below.